packages/mui-base/src/MenuUnstyled/MenuUnstyled.test.tsx

```diff
@@ -37,6 +38,39 @@ describe('MenuUnstyled', () => {
     skip: ['reactTestRenderer', 'propsSpread', 'componentProp', 'slotsProp'],
   }));
 
+  describe('after initialization', () => {
+    const spyFocus = spy();
+
+    function Test() {
+      React.useEffect(() => {
+        document.addEventListener('focus', spyFocus, true);
+        return () => {
+          document.removeEventListener('focus', spyFocus, true);
+        };
+      }, []);
+
+      return (
+        <MenuUnstyled {...defaultProps}>
+          <MenuItemUnstyled data-testid="item-1">1</MenuItemUnstyled>
+          <MenuItemUnstyled data-testid="item-2">2</MenuItemUnstyled>
+          <MenuItemUnstyled data-testid="item-3">3</MenuItemUnstyled>
+        </MenuUnstyled>
+      );
+    }
+
+    it('when menu is opened it highlights one item and it must be the first one', () => {
+      const { getAllByTestId } = render(<Test />);
+      const [firstItem, ...otherItems] = getAllByTestId(/^item-/);
```

Comment:
No need to introduce test ids here:
```suggestion
        <MenuUnstyled {...defaultProps}>
          <MenuItemUnstyled>1</MenuItemUnstyled>
          <MenuItemUnstyled>2</MenuItemUnstyled>
          <MenuItemUnstyled>3</MenuItemUnstyled>
        </MenuUnstyled>
      );
    }

    it('highlights the first item when the menu is opened', () => {
      const { getAllByRole } = render(<Test />);
      const [firstItem, ...otherItems] = getAllByRole ('menuitem');
```

Reply:
Done i changed it thanks 👍